### PR TITLE
SB-25552 - UCI Transformer Deployment

### PR DESCRIPTION
**Pull request by shiva-rakshith**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-25552" title="SB-25552" target="_blank">StorySB-25552</a>  UCI - Deployment
  </td></table>
  <br />
 

uci transformer deployment config

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by shiva-rakshith** [comment]
@keshavprasadms Please review the PR.

File: kubernetes/helm_charts/core/content/values.j2 [hunk 1]
```diff
@@ -5,31 +5,31 @@ imagepullsecrets: {{ imagepullsecrets }}
 dockerhub: {{ dockerhub }}
 
 env:
-  javaoptions: {{content_java_mem_limit|default('-Xmx600m')}}
+  javaoptions: {{uci_outbound_java_mem_limit|default('-Xmx600m')}}
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/core/content/values.j2:8]: Why are we editing `kubernetes/helm_charts/core/content/values.j2`?

Comment (shiva-rakshith) [reply kubernetes/helm_charts/core/content/values.j2]: Changes were made to wrong file, reverted the changes.

File: ansible/roles/stack-sunbird/templates/transformer_industry_feedback.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...quirements/average_hiring" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/current_requirement_number" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/time_frame" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/hiring_local" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/hiring_apprentices" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/apprentices_number" relevant=" /data/hiring_requirements/hiring_apprentices = 'Yes'" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/usage_rozgar" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/reason_rozgar_usage" relevant=" /data/hiring_requirements/usage_rozgar = 'No'" required="true()" type="string"/><bind jr:preload="uid" nodeset="/data/meta/instanceID" readonly="true()" type="string"/></model></h:head><h:body><group ref="/data/personal_details"><label>Introduction</label><input ref="/data/personal_details/form_intro"><label>- This survey is being conducted by the Saksham Haryana team, CM Office in light of the current labour crisis being faced by establishments.
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/transformer_industry_feedback.xml:1]: This file is impossible to read. Can we format it to make it more readable? https://jsonformatter.org/xml-formatter

Comment (shiva-rakshith) [reply ansible/roles/stack-sunbird/templates/transformer_industry_feedback.xml]: Converted the file content into proper XML format.

**Comment by keshavprasadms** [comment]
@shiva-rakshith Please resolve conflicts

**Comment by shiva-rakshith** [comment]
> @shiva-rakshith Please resolve conflicts

@keshavprasadms If we close the #2687, again we will get new conflicts, So waiting for PR to be merged, so that we can resolve conflicts here.

**Comment by keshavprasadms** [comment]
Done. Please check conflicts now

**Comment by shiva-rakshith** [comment]
@keshavprasadms Resolved conflicts.

**Review by keshavprasadms — APPROVED** [review]
LGTM

**Pull request merged**